homework-2/scss/style.scss

```diff
@@ -0,0 +1,1385 @@
+  width: 35px;
+  height: 35px;
+  background: transparent;
+  border: 2px solid #22bf95;
+  border-radius: 50%;
+  &::before{
+    content: "";
+    width: 2px;
+    background: #22bf95;
+    height: 9px;
+    position: absolute;
+    top: 14px;
+    transform: rotate(138deg);
+    left: 9px;
+  }
+  &::after{
+    content: "";
+    width: 2px;
+    background: #22bf95;
+    height: 18px;
+    position: absolute;
+    top: 7px;
+    transform: rotate(-138deg);
+    left: 18px;
+  }
+}
+.minus-icon{
+  position: relative;
+  display: inline-block;
+  width: 35px;
+  height: 35px;
+  background: transparent;
+  border: 2px solid #e5e5e5;
+  border-radius: 50%;
+  &::before{
+    content: "";
+    width: 18px;
+    background: #e5e5e5;
+    height: 2px;
+    position: absolute;
+    top: 14px;
+    left: 8px;
+  }
+}
+.arrow{
+  height: 13px;
+  position: relative;
+  width: 1px;
+  background: #fff;
+  margin: 4px 16px 0 100px;
+  display: inline-block;
+  border: 1px solid #fff;
+  &:before{
+    content: "";
+    width: 2px;
+    background: #fff;
+    height: 10px;
+    position: absolute;
+    top: -5px;
+    transform: rotate(50deg);
+    left: -4px;
+  }
+  &:after{
+    content: "";
+    width: 2px;
+    background: #fff;
+    height: 10px;
+    position: absolute;
+    top: -5px;
+    transform: rotate(-50deg);
+    right: -4px;
+  }
+}
+.right{
+  transform: rotate(90deg);
+}
+.footer{
+  background-color: #f2f2f2;
+  padding: 30px 0;
+  &-item-inner{
+    padding: 8px 0 8px 0;
+    border-top: 2px solid #22bf95;
+  }
+  &-title {
+    font-size: 16px;
+    line-height: 36px;
+    color: #7f7f7f;
+  }
+  &-list{
+    max-width: 176px;
+    width: 100%;
+  }
+}
+.menu-footer-list{
+  padding-left: 23px;
+  & li + li{
+    padding-top: 10px;
+  }
+  .menu-link{
+    display: block;
+    padding: 8px 0;
+    color: #7f7f7f;
+    font-family: "Acme-Regular", sans-serif;
+    font-size: 12px;
+    line-height: 24px;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+  }
+}
+.phone{
+  margin-top: 4px;
+  margin-bottom: 10px;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f7f7f;
+  transition: 1s;
+  &:hover{
+    color: #22bf95;
+  }
+}
+address{
+  font-size: 12px;
+  line-height: 20px;
+  color: #7f7f7f;
+}
+.link-map{
+  font-family: "Aclonica-Regular", sans-serif;
+  font-size: 12px;
+  line-height: 20px;
+  color: #22bf95;
+  transition: 1s;
+  &:hover{
+    color: #7f7f7f;
+    text-decoration: underline;
+  }
+}
+.social{
+  padding-bottom: 24px;
+  &-inner{
+    border-top: 2px solid #b2b2b2;
+  }
+  &-list{
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    margin: 29px 14px;
+  }
+}
+svg .social-hover{
+  transition: 200ms;
+  transition-timing-function: ease-in-out;
+  -webkit-transition: 200ms;
+  -webkit-transition-timing-function: ease-in-out;
+}
+svg:hover .social-hover{
+  fill: #22bf95;
+}
+.footer-logo-img{
+  display: block;
+  margin: 0 auto;
+}
+@include tablet{
```

Comment:
Better to use media-queries right inside block. For example:
```
.container {
  padding: 0 8px;

  @media screen and (min-width: 768px) {
    padding: 0 16px;
  }

  @media screen and (min-width: 1200px) {
    padding: 0 32px;
  }
}
```

It is very difficult to understand code when styles for same element are in several different places.

Reply:
done